app/views/shared/_articles.html.erb

```diff
@@ -28,7 +28,11 @@
 
     <articles>
       <% @articles.each_with_index do |article, index| %>
-       <%= render partial: 'shared/article', locals: { article: article, index: index } %>
+
+        <% if article.layout_id == 4 # check that the article layout is being used %>
```

Comment:
Is there any other way to check for the right layout instead of using the id?
The id might potentially change if we remove and recreate the layout for example.
It would be good to have something like name, or slug